qemu/tests/cfg/blockdev_mirror_to_rbd.cfg

```diff
@@ -0,0 +1,29 @@
+- blockdev_mirror_to_rbd:
+    only Linux
+    start_vm = no
+    qemu_force_use_drive_expression = no
+    type = blockdev_mirror_to_rbd
+    virt_test_type = qemu
+    images += " data1"
+    source_images = data1
+    target_images = mirror1
+    remove_image_data1 = yes
+    force_create_image_data1 = yes
+    backup_options_data1 = sync
+    sync = full
+    storage_pools = default
+    storage_pool = default
+
+    image_size_data1 = 2G
+    image_size_mirror1 = ${image_size_data1}
+    image_format_mirror1 = raw
+    image_name_data1 = data1
+    image_name_mirror1 = mirror1
+    storage_pool_mirror1 = rbd
+    rbd_pool_name_mirror1 = ${storage_pool_mirror1}
+    storage_type_rbd = ceph
+    rebase_mode = unsafe
+
+    # For local mirror images
+    storage_type_default = directory
+    enable_ceph_mirror1 = yes
```

Comment:
enable_ceph_data1 = no
enable_ceph_image1 = no

Disable ceph storage backend for data1 and image1, which are still using the local image

Reply:
Have updated, thanks zhenchao